meteorite_fall.js

```diff
@@ -51,6 +62,12 @@ function initializeMeteorites() {
 
 // key events
 document.addEventListener('keydown', (e) => {
+    if (e.key == "p")
+        paused = !paused;
+
+    if (paused)
```

Comment:
I commented this out and it doesn't affect the functionality - game is still pausing as needed. Is it possible to go without this if-statement?

Reply:
It does. If not then they can still move the player left and right while the game is paused lol. Try it

Reply:
not being mean

Reply:
I clean upped the code in the mainloop func, and was able to remove this actually.